### PR TITLE
Added thoughts on code review

### PR DESCRIPTION
**Pull request by toddlamothe**
## Summary
Inspired by an article @rallytime shared, thoughts on code reviews and the importance of well written pull requests

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,6 +15,7 @@
 - use intention revealing names
 - write self documenting code
 - don't leave [broken windows](https://blog.codinghorror.com/the-broken-window-theory/) unrepaired
+- Pending code reviews represent blocked threads of execution. To prevent getting blocked for more time than is necessary, [make your pull requests a pleasure to read](https://medium.com/@9len/on-code-review-16ea85f7c585)
```
[the remark this answers is not in the document]
Comment (toddlamothe) [reply README.md]: I split code review into its own section and added separate bullet points as suggested

It's still a bit wordy - better than before?

[the remark this answers is not in the document]
Comment (smaraf) [reply README.md]: I love it!

**Review by smaraf — APPROVED** [review]
👍  Nice addition!
